glide.yaml

```diff
@@ -16,6 +16,12 @@ import:
   version: v0.0.1
 - package: github.com/spf13/pflag
   version: v1.0.0
+- package: github.com/openshift/api
+- package: github.com/fatih/color
+  version: v1.6.0
+- package: github.com/docker/docker
```

Comment:
Do we really need it? It worked fine for me without this.

Reply:
I'll remove it and try another version of glide, if it doesn't work I'll add an issue upstream!